package.json

```diff
@@ -21,7 +21,7 @@
     "bowser": "1.6.0",
     "crypto-md5": "^1.0.0",
     "debug": "4.3.1",
-    "esotope-hammerhead": "0.6.4",
+    "esotope-hammerhead": "https://github.com/brandwatch/esotope-hammerhead.git#feat-static-block",
```

Comment:
I published esotope-hammerhead with your fix. Please update the esotope-hammerhead version here